src/pages/dashboard.js

```diff
@@ -1,153 +0,0 @@
-import React, { useEffect, useState } from 'react';
-import {Alert, Box, Container, Grid} from '@mui/material';
-
-import ViewListIcon from "@mui/icons-material/ViewList";
-import BitcoinIcon from '@mui/icons-material/CurrencyBitcoin';
-import LocationSearchingIcon from '@mui/icons-material/LocationSearching';
-import PaymailIcon from '@mui/icons-material/Message';
-
-import { AdminChart } from '../components/dashboard/chart';
```

Comment:
I assume that we have some dependencies in package.json that were added only to display a chart,
please ensure that they are removed if not used after deletion of this component.

Reply:
Removed chart-js dependency